templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -45,7 +45,7 @@
               {/block}
 
               {block name='cart_voucher_notifications'}
-                <div class="alert alert-danger js-error d-none" role="alert">
+                <div class="alert alert-danger js-error mt-2" role="alert" style="display: none;">
```

Comment:
Inline style? 🤔

Reply:
@kpodemski this is a core limitation from the `core.js` file toggling this error when the voucher trigger it